<?xml version="1.0" encoding="UTF-8"?><testrun name="AppTest" project="assingment1" tests="2" started="2" failures="0" errors="1" ignored="0">
  <testsuite name="rahul.assingment1.AppTest" time="318.297">
    <testcase name="testApp" classname="rahul.assingment1.AppTest" time="0.0"/>
    <testcase name="testOrder" classname="rahul.assingment1.AppTest" time="318.297">
      <error>java.lang.NullPointerException
	at rahul.assingment1.validators.IValidate.validateOrdering(IValidate.java:114)
	at rahul.assingment1.AppTest.lambda$1(AppTest.java:73)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at rahul.assingment1.AppTest.testOrder(AppTest.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:121)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</error>
    </testcase>
  </testsuite>
</testrun>
